index.html

```diff
@@ -25,7 +25,7 @@ <h1 class="title">Weather App</h1>
 
           <h2 class="results-city"></h2>
 
-
+           <div class=iconPic></div>
```

Comment:
Try to stick with consistent naming conventions. All other class names are written with a hyphen so this should follow that pattern and be "icon-pic".